resources/js/Pages/Praktikan.vue

```diff
@@ -906,19 +983,38 @@
           </span>
         </div>
 
+        <div class="h-full flex animation-enable pointer-events-none"
+            :class="[{ 'w-1/11': !isJawaban },
+                    { 'w-9/11': isJawaban }]"
+            v-on:click="showJawaban">
+          <div class="h-full flex animation-enable pointer-events-none"
+              :class="[{ 'w-full': !isJawaban },
+                      { 'w-1/2': isJawaban }]">
+            <div class="h-full animation-enable pointer-events-none"
+              :class="[{ 'w-0': !isJawaban },
+                      { 'w-9/12': isJawaban }]"/>
+            <img class="jawabanIcon w-full iconYellowHover select-none cursor-pointer pointer-events-auto self-center h-8 fas fa-book animation-enable">
+          </div>
+          <span class="self-center text-left font-monda-bold text-lg text-white animation-enable-short"
+              :class="[{ 'w-0 opacity-0 tracking-tighter': !isJawaban },
+                      { 'w-1/2 opacity-100 tracking-widest': isJawaban }]">
+            Jawaban
+          </span>
+        </div>
+
```

Comment:
remove this commented code, remember you always use git for this repo, don't worry if u forget this code 😉

Reply:
Eh, I didn't comment this code though?  because this is the main code in menu list for praktikan. In my editor it's not commented but in github it's commented. is it because `<!------------------------->` at line 908 and 1018?